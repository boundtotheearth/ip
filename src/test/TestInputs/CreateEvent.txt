event go home /at 2020-10-10
list
