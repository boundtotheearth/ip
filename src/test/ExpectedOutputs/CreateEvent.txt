Added: [E][✘] go home (at: 10 Oct 2020)
1. [E][✘] go home (at: 10 Oct 2020)
